crates/noirc_frontend/src/node_interner.rs

```diff
@@ -179,6 +197,7 @@ enum Node {
     Function(HirFunction),
     Statement(HirStatement),
     Expression(HirExpression),
+    TypeAlias,
 }
 
```

Comment:
It can be removed - we can handle it the same way we do with struct definitions. They don't have a corresponding Statement/HirStatement since they're unneeded after name resolution remembers them in the interner.